### ISSUE TITLE
Game Simulation not ticking since 0.9.3

### ISSUE THREAD

**Issue by Myrc** [StrangeLoopGames/EcoIssues#21431]
We have a server started on [Eco Server 0.9.2.4 beta release-156] where everything worked as it should but the game simulation has not been ticking since 9.3, this is still an issue in "Eco Server 0.9.3.1 beta release-204"

The obvious symptoms are that the plant has not been growing and that tailings don't pollute. I have tried planting and waiting for a few hours as well as using /ff with 0 growth according to the visuals and the soil tool.

The server was started with a few mods that didn't cause any issues in 9.2. (big shovel, color vehicles, discord link and nidtoolbox) none of these should do anything with the game simulation, to be sure we have removed all of the mods and yet the problem persists.

The server is hosted by a 3d party "verygames.net"

There's is a log warning "interaction distance check failed (occupancy based)" on carts buts that's likely not pertinent to the sim ticks

<img width="345" alt="1" src="https://user-images.githubusercontent.com/4669107/115095683-707ee180-9f22-11eb-9fb3-f73f2dd4ecf0.PNG">
<img width="227" alt="2" src="https://user-images.githubusercontent.com/4669107/115095685-71177800-9f22-11eb-8485-ec70da7f0b92.PNG">

[log_210417125126.log](https://github.com/StrangeLoopGames/EcoIssues/files/6328328/log_210417125126.log)

[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6328373/Player.log)

The server directory was too large to attach but if wanted i can find a way to give it to you.

**Comment by SlayksWood** [comment]
@Myrc Thank you for the report. Can you provide save files?

**Comment by IcedForge** [comment]
Please reopen this issue if this is still occurring for you. Will close for now in lack of reply after requesting additional information.

**Comment by legionaw** [comment]
Our server is also having this issue in which plants are not growing or spawning at all. The only common thread I can see here in this between our server and the OP's server based on description above is that we have the first three of mods listed (big shovel, color vehicles, and DiscordLink) but, as OP pointed out, it is very unlikely that it would have any impact on plant growth or simulation itself. I should note that this server was started after v0.9.3 was released for Eco.

I am not sure if this other issue is connected to this, but even couple weeks (in real time) after I removed tailings waste from underground storage near river mouth, ground pollution that resulted from this now-defunct storage did not completely dissipated. But it's gone now with /regenlayer GroundPollutionSpread so I guess that's fixed now? It is probably likely that this is unrelated but I can't help but wonder if plant growth and pollution aren't updating or something. I haven't checked to see if the map showed any new pollution recently but I can't say I remember any ground pollution near sewage output now that I think about it.

I also don't know if this is related or not but server for some reason refuse to shut down even after I pressed Ctrl+C, with only log console remaining while the settings console did closed, still doing automatic backups if left on for quite a while. As a result, I was forced to right-click log console's taskbar icon and click "Close window" which did the trick after I press Ctrl+C. This happen every time. This never happened to me before 9.3 came out.

For some reason, I am unable to attach save file here, as GitHub is giving me an error saying "is not included in the list" in red font, whatever that mean. So I am linking it from my Google Drive instead [here](https://drive.google.com/file/d/15jkw2gKwh5WsvvushQCIy3HzCJT7QAgO/view?usp=sharing).

**Comment by mirasrael** [comment]
@legionaw loaded your save on latest release version and it correctly ticks, shutdowns and spawn plants. Can you try to load your save on fresh server installation?

**Comment by legionaw** [comment]
> 
> 
> @legionaw loaded your save on latest release version and it correctly ticks, shutdowns and spawn plants. Can you try to load your save on fresh server installation?

Well, I discovered that it did not ticked at all, showing 0 ticks in World Layers tab on log console so now the question why it didn't tick at all.

As for fresh server install, can you clarify? Do I just uninstall and then reinstall? I already revalidated the install on Steam but that did not help.

**Comment by mirasrael** [comment]
Try install to different folder. Problem something wrong with you configs/mods.

**Comment by legionaw** [comment]
> 
> 
> Try install to different folder. Probably something wrong with your configs/mods.

Well, I tested with a new world with same seed, on same install path (post-reinstall), with same mods, and the plant growth is ticking OK. Unsure why or what caused it in original world but I am guessing there is something corrupt within the original world's save file. :X However, that doesn't explain why you were able to get plant growth to tick again in that world save on your end.

It should be noted that original world was already tested without any mods before this new world testing and the ticking issue persisted. Furthermore, I should clarify that neither Simulation nor WorldLayer configurations for original world have been modified in any way at any time, except for modifying the Oilfield world layer in latter but that modification was made AFTER this issue had already appeared.

In fact, I am attaching an image showing that these are the only configurations that have been modified via console from its default parameters based on templates and none of these seem likely to be a culprit for this. Perhaps except for WorldGenerator.eco but that seem extremely unlikely as that is obviously concerned with world generation and would probably not have any effect post-generation.

![explorer_2021-07-12_168](https://user-images.githubusercontent.com/4440182/125382364-29ef2680-e35b-11eb-9125-0cbf28c55944.png)

At any rate, we are planning to generate a new world anyway and moving almost every buildings to it that we have saved via the new blueprint function from WorldEdit mod, while adminstratively set the spec and level to what it had been in original world. It was going to be done anyway as we wanted a world with wider river and such. So it's all good now. I guess this is a mystery that will have to remain unresolved for now.
